src/tutorial/microbe_editor/NucleusTutorial.cs

```diff
@@ -0,0 +1,114 @@
+﻿namespace Tutorial;
+
+using System;
+using Newtonsoft.Json;
+
+/// <summary>
+///   Prompts the player to eventually place a nucleus for further progression
+/// </summary>
+public class NucleusTutorial : TutorialPhase
+{
+    private const int TriggersOnNthEditorSession = 11;
+
+    private readonly string cellEditorTab = EditorTab.CellEditor.ToString();
+
+    private bool hasNucleus;
```

Comment:
This has to be JsonProperty otherwise the game will forget if the player has a nucleus or not when they save/load.